### PR TITLE
2020-04 updates to public website

### LINKED ISSUE #88: Remove mention of md5 checksums from BagIt Spec and profile
## Is your feature request related to a problem? Please describe.
Aurora is now running on CentOS 8, which does not support md5 checksums for security reasons. 

## Describe the solution you'd like
Remove all mentions of md5 checksums from the RAC BagIt spec as well as the sample BagIt Profile.

### PR DESCRIPTION
**Pull request by HaSistrunk**
Fixes #88 , #89 , and #90 

- Minor updates to team bio descriptions
- Adds RFCs 005 (arrangement maps) and 006 (request brokers)
- Edit RAC BagIt Spec to remove md5 as an accepted 
- Edit `organizational-bag-profile.json` [gist](https://gist.github.com/HaSistrunk/65d59e558c436b9d934d98fd8fb0f575) shown on Scripts page to include `Manifests-Allowed` key specifying sha256 and sha512 and remove `Manifests-Required`.
- Changes layout of Components page to replace cards with lists
- Updates Components page to include all Project Electron repositories and their short descriptions (except Argo)

### CODE REVIEW

File: content/components.md [hunk 1]
```diff
@@ -9,74 +9,40 @@ Project Electron consists of a specification and web application to facilitate o
...
-    <p>An API gateway and administration interface for Project Electron microservices, managed via a message queue</p>
-  </div>
-</div>
+  - [zodiac](https://github.com/RockefellerArchiveCenter/zodiac): an API gateway and administration interface for Project Electron microservices, managed via a message queue.
+  - [ursa major](https://github.com/RockefellerArchiveCenter/ursa_major): discovers and stores bags.
+  - [fornax](https://github.com/RockefellerArchiveCenter/fornax): creates Archivematica-compliant Submission Information Packages.
+  - [gemini](https://github.com/RockefellerArchiveCenter/gemini): downloads packages from the Archivematica Storage Servicestores and stores them in Fedora.
+  - [aquarius](https://github.com/RockefellerArchiveCenter/aquarius): transforms and delivers accessions, archival objects and digital objects to ArchivesSpace.
+  - [libra](https://github.com/RockefellerArchiveCenter/libra): generates reports on files and activities in Fedora.
+  - [zorya](https://github.com/RockefellerArchiveCenter/zorya): packages bags.
```
Comment (helrond) [inline content/components.md]:
```suggestion
  - [zorya](https://github.com/RockefellerArchiveCenter/zorya): creates bags from files generated by disk imaging and digitization processes.
```

File: content/components.md [hunk 2]
```diff
@@ -9,74 +9,40 @@ Project Electron consists of a specification and web application to facilitate o
...
+### Discovery
 
-  <div class= "six columns card">
-    <h3>
-      <a href="https://github.com/RockefellerArchiveCenter/fornax">fornax</a>
-    </h3>
-    <p>Creates Archivematica-compliant Submission Information Packages</p>
-  </div>
-</div>
+- [pisces](https://github.com/RockefellerArchiveCenter/pisces): retrieves and transforms data for discovery.
```
Comment (helrond) [inline content/components.md]:
```suggestion
- [pisces](https://github.com/RockefellerArchiveCenter/pisces): retrieves, merges and transforms data for discovery.
```

File: content/components.md [hunk 3]
```diff
@@ -9,74 +9,40 @@ Project Electron consists of a specification and web application to facilitate o
...
+- [scorpio](https://github.com/RockefellerArchiveCenter/scorpio): merges, indexes, and deletes archival data (collections, objects, agents and terms).
```
Comment (helrond) [inline content/components.md]:
```suggestion
- [scorpio](https://github.com/RockefellerArchiveCenter/scorpio): adds and deletes archival data (collections, objects, agents and terms) from an index.
```

**Review by helrond — CHANGES_REQUESTED** [review]
A couple of minor things...I'm going to look at the short description for those repositories right now.

**Comment by HaSistrunk** [comment]
Ok great, thanks. I didn't include Argo here, but it does have topic tags. Do you think that's ok @helrond ?

**Comment by helrond** [comment]
@HaSistrunk I think that's okay for now - it's in a pretty early state.

**Review by helrond — APPROVED** [review]

**Pull request merged**